src/components/Progress/sgds-progress-bar.ts

```diff
@@ -0,0 +1,75 @@
+import { property } from "lit/decorators.js";
+import { styleMap } from "lit/directives/style-map.js";
+import { classMap } from "lit/directives/class-map.js";
+import SgdsElement from "../../base/sgds-element";
+import styles from "./sgds-progress-bar.scss";
+import { html } from "lit";
+
+/**
+ * @csspart label - Progress bar label
```

Comment:
there is no part=label in the html below

Reply:
resolved in [bc37b42](https://github.com/GovTechSG/sgds-web-component/pull/110/commits/bc37b42418386e8c82ebece48dbf312fda680f2e)